packages/dashboard/.eslintrc.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
     },
     rules: {
         'import/no-extraneous-dependencies': 'off',
+        "@typescript-eslint/no-explicit-any": "off",
```

Comment:
Instead of disabling the rule, why not enforce developers to type stuff properly? In the case we're using an external value of which we don't know the type, we can always use `unknown` or ignore that line specifically. This will make developers lazy and they'll get away with it.